package.json

```diff
@@ -223,7 +223,7 @@
     "rollup-plugin-url": "^3.0.1",
     "rollup-plugin-visualizer": "^4.2.0",
     "semantic-release": "^19.0.5",
-    "stream-chat": "^8.15.0",
+    "stream-chat": "^8.21.0",
```

Comment:
Shouldn't you also bump the peer dependency?

Reply:
if we keep it at `^8.0.0` - this means our SDK works with `stream-chat: 8.0.0` without any issues - and I'm pretty sure that's not the case :)

Reply:
bumped